staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/metrics.go

```diff
@@ -310,6 +310,19 @@ var (
 		},
 		[]string{priorityLevel, "success"},
 	)
+	apiserverWorkEstimateSeats = compbasemetrics.NewHistogramVec(
```

Comment:
@MikeSpreitzer this metric may be redundant?

Reply:
Do you mean with the watch count metric? I am considering removing that one.  The metric introduced in this PR has the virtue that it measures something closer to the scheduling mechanism.

The object count metric is labeled by resource rather than APF classification.

Reply:
I agree - if removing anything, then not this one - this one actually translates to something user-visible, so I claim it's useful.